source/common/config/grpc_mux_impl.cc

```diff
@@ -222,10 +224,17 @@ void GrpcMuxImpl::onStreamEstablished() {
 }
 
 void GrpcMuxImpl::onEstablishmentFailure() {
-  for (const auto& api_state : api_state_) {
-    for (auto watch : api_state.second.watches_) {
-      watch->callbacks_.onConfigUpdateFailed(
-          Envoy::Config::ConfigUpdateFailureReason::ConnectionFailure, nullptr);
+  // Try to connect all of subscribed clusters which are included on grpc_service. If all of
```

Comment:
This comment is a bit vague. I think you're trying to suggest that we're actually going to pick the gRPC service in the event of failure and try to connect.